roles/manage-ec2-instances/tasks/provision.yml

```diff
@@ -647,7 +647,7 @@
     src: instructor_inventory.j2
     dest: "{{ playbook_dir }}/workshops/{{ name_prefix }}/instructor_inventory.txt"
 
-- name: Generate student inventories
+- name: Generate student inventories 2
```

Comment:
Why is this being changed?  Also, the commit for this says that gitlab needs more RAM, but that isn't what is being changed.

Reply:
must be a typo, i put in another commit to undo this